cli/main.go

```diff
@@ -53,7 +54,12 @@ func main() {
 		&cobra.Command{
 			Use:   "add-namespace",
 			Short: "Usage: <namespace> <owner>",
+			Args:  cobra.MinimumNArgs(2),
 			RunE: func(cmd *cobra.Command, args []string) error {
+				if len(args) == 2 {
+					args = append(args, "")
+				}
```

Comment:
This looks like a dirty hack. Why?

Reply:
because of the `docker-compose exec cli ./cli add-namespace <namespace> <owner>` command that should not be used unless you know what you are doing, this avoids creating a panic when `tenant_id` is not informed.

Reply:
So please add a clear log explaining this.

Reply:
Sure, i'll do it